mmv1/templates/terraform/resource.go.tmpl

```diff
@@ -722,9 +722,7 @@ func resource{{ $.ResourceName -}}Read(d *schema.ResourceData, meta interface{})
     {{ $.CustomTemplate $.CustomCode.CustomIdentityRead true -}}
 {{- else }}
 	identity, err := d.Identity()
-	if err != nil {
-		return fmt.Errorf("Error getting identity: %s", err)
-	}
+    if identity != nil {
```

Comment:
should we be checking for `err != nil && identity != nil` ?

Also looks like the indentation is now off if we are wrapping the next few lines in a new block. Formatting gets this later, but is a little more confusing in the template.